scripts/deploy.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+# Color definitions
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+
+# Check if version number is provided
+if [ "$#" -lt 1 ]; then
+    echo -e "${BLUE}Usage: $0 <version_number> [commit_message] [--deployer=NAME] [--exclude-release] [--no-translations] [--mainnet] [--deploy-only]${NC}"
+    echo
+    echo -e "${CYAN}Examples:${NC}"
+    echo "1. Simple message (testnet):"
+    echo -e "   ${GREEN}$0 3.0.10 'Fix network stall handling' --deployer=matheo${NC}"
+    echo
+    echo "2. Multi-line message (mainnet):"
+    echo -e "   ${GREEN}$0 3.0.4 '- Move network browsers-not-shown warning to not overlap with bottom row"
+    echo "- Add a \"Failed to fetch transactions\" notice when transaction fetching fails"
+    echo "- Add a retry-mechanism to all Nimiq network requests"
+    echo "- Also hide staked amounts when privacy mode is on"
+    echo -e "- Enforce minimum stake on the slider itself' --deployer=john --mainnet${NC}"
+    echo
+    echo "3. Deploy only (after a cancelled deployment):"
+    echo -e "   ${GREEN}$0 --deploy-only${NC}"
+    exit 1
+fi
+
+# Don't require version number if --deploy-only is specified
+if [[ "$1" == "--deploy-only" ]]; then
+    VERSION=""
+    DEPLOY_ONLY=true
+    shift
+else
+    VERSION="$1"
+    shift # Remove version from arguments
+fi
+
+# Default values
+DEPLOYER="matheo"
+EXCLUDE_RELEASE=""
+SYNC_TRANSLATIONS=true
+BUILD_ENV="testnet"
+DEPLOY_ONLY=false
```

Comment:
DEPLOYER probably shouldnt' have a default ;)

And I would also like BUILD_ENV not to have a default, but to always require explicitly specifying it on the command line.

Reply:
that's true! I wanted to remove it and forgot, my bad! 👍

Reply:
DEPLOYER is now required, with no default value.

I also added a `--testnet` parameter, and made it required to have at least either `--testnet` or `--mainnet`.